services/QuillLMS/app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-
+  QUILL_TEAM_EMAIL_ADDRESS = 'hello@quill.org'
```

Comment:
Probably not worth futzing with in this PR, but we may want to set up some specific email address for this kind of issue.

Reply:
While I do agree with the smell of using the same email everywhere, Stripe subscriptions has been on since May and we've only had one of these errors, so the volume should be very low.